server/src/internalClusterTest/java/org/opensearch/ingest/IngestClientIT.java

```diff
@@ -177,6 +188,9 @@ public void testBulkWithIngestFailures() throws Exception {
 
         int numRequests = scaledRandomIntBetween(32, 128);
         BulkRequest bulkRequest = new BulkRequest();
+        if (shouldSetBatchSize) {
+            bulkRequest.batchSize(numRequests);
```

Comment:
How about:
```suggestion
            bulkRequest.batchSize(scaledRandomIntBetween(2, numRequests));
```

Can you run this in a loop a ton of times as well just to make sure there's not a lingering bug with any combination of batch size relative to bulk size?

Reply:
Changed to suggested code locally and ran test with this command 
```
./gradlew :server:internalClusterTest -Dtests.iters=300 --tests "org.opensearch.ingest.IngestClientIT.testBulkWithIngestFailuresWithBatchSize"
```
All passed.